source/documentation/deploying_services/database_services.md

```diff
@@ -47,37 +51,47 @@ See the [Amazon RDS documentation on the failover process](http://docs.aws.amazo
 
 You should test how your app deals with a failover to make sure you are benefiting from the high availability plan. We can trigger a failover for you. Please contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) to arrange this.
 
-### Encrypted PostgreSQL plans
+#### High availability plans - MongoDB
+
+MongoDB uses replica sets i.e. clusters of MongoDB servers that replicate the contents of the master database. Replica sets provide high availability through sharding, a method for distributing data across multiple machines. A shard is a single mongod instance or replica set that stores some portion of a sharded cluster’s total data set. In the standard configuration for MongoDB on Compose, you get a single-shard with three nodes in a replica-set. Mongo will push all write operations on to the primary data node and then propagate them to the secondary data node on the shard. Should the primary node fail, the secondary will be promoted to primary providing fault-tolerance and redundancy for your databases. The Mongo routers are responsible for dictating any failover, along with load-balancing operations to any of the nodes in the cluster.
```

Comment:
In the postgres docs we do mention that it's provided by RDS. This PR removes that, which is not useful to the user IMO. If we're clear about the underlying technology then users can find out more information about them - anything we haven't documented.

But I think this is probably not the correct place for it, the correct place is at the beginning of the Mongo docs